src/components/page/Header/index.tsx

```diff
@@ -238,7 +238,13 @@ export class HeaderComponent extends React.Component<HeaderProps, State> {
                 </UncontrolledDropdown>
               )}
 
-              {(ENABLE_TEAMS || ENABLE_PRACTITIONERS || ENABLE_USERS || ENABLE_MDA_POINT) && (
+              {(ENABLE_TEAMS ||
```

Comment:
Does it make sense to now create a variable to hold this value?

Reply:
yeap